packages/@aws-cdk/aws-cognito-identitypool/lib/identitypool-user-pool-authentication-provider.ts

```diff
@@ -100,19 +99,36 @@ export class UserPoolAuthenticationProvider implements IUserPoolAuthenticationPr
   }
 
   public bind(
-    scope: Construct,
```

Comment:
I don't think this will work if you remove this. Without providing the construct, `this` is meaningless.

Reply:
> Without providing the construct, `this` is meaningless.

Can you expand on this?  I'm familiar with the [`.bind()` method on a function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Function/bind), but this is a method on a class. To my knowledge (which very well could be missing something), `UserPoolAuthenticationProvider.bind()` has no special relationship with traditional binding beyond sharing the same name. If `super.bind()` were called, I would re-evaluate that last statement but at this time I believe it's correct.  Looking at the method, we can see that `scope` is unused, making me feel that it is safe to remove.   Reviewing the codebase, we can find other instances of `bind()` where a `scope` is not passed in but `this` is used:

https://github.com/aws/aws-cdk/blob/b8708e9ad476d12d0654ce34ae296f9c737c804d/packages/%40aws-cdk/aws-lambda-event-sources/lib/s3.ts#L28-L33